src/util/path.js

```diff
@@ -590,6 +590,13 @@
         currentPath,
         parsed,
         re = fabric.rePathCommand,
+        rCommaWsp = '(?:\\s,?\\s*|,\\s*)',
```

Comment:
we have this same string here in parser.js.
What about making it exactly as fabric.reNum and change the line
https://github.com/fabricjs/fabric.js/blob/master/src/parser.js#L249
with
```js
commaWsp = fabric.commaWsp,
```

Here we can do the same, also for fabric.reNum, if we put it in its own var the minifier will save something.

Reply:
Actually, these aren't identical. Yours matches the grammar better. 💯
Changing up the three files.